README.md

```diff
@@ -1,8 +1,8 @@
 # Startups guide to Azure
-This repo provides startups with Azure guidance, information, and resources. Whether you're creating something brand new, you're considering a move from another cloud provider, you want to leverage Azure services, you're refactoring your architecture, or you're throwing everything away and starting over, this guide will help you.
+This repo aims to provide you, a startup founder/technical decision maker/developer/curious learner with Azure guidance, information, and resources. Whether you're creating something brand new, you're considering a move from another cloud provider, you want to leverage Azure services, you're refactoring your architecture, or you're throwing everything away and starting over, this guide will help you.
```

Comment:
How about "This repo provides you with Azure guidance...." That way we don't define a specific audience. Thoughts/

Reply:
Yea, that's good. Although, I do like the thinking of what do small agile companies need to understand quickly and saying that upfront. But, this is elegant and solves it.

Reply:
Will you update the text in your repo and commit? The PR will automatically update.